<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab works #3 &amp; #4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	background:qlineargradient(spread:reflect, x1:0.506, y1:0.506136, x2:0.505682, y2:1, stop:0 rgba(144, 194, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: gray;
    padding: 5px;
}
QPushButton:pressed
{
	background: qlineargradient(spread:reflect, x1:0.506, y1:0.506136, x2:0.511364, y2:0.989, stop:0 rgba(85, 85, 255, 255), stop:1 rgba(144, 194, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="graphLayout" stretch="0,0">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Graph Output</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="LeftLayout" stretch="0,0,0">
      <item>
       <layout class="QVBoxLayout" name="Matrlayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Matrix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="incEdit">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="resultGroup">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="resultLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Result</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="Deicstra">
             <property name="text">
              <string>Deicstra's Algorithm</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Ford">
             <property name="text">
              <string>Ford-Falkerson's Algorithm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="ResultEd">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="resTableWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>83</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="rowCount">
            <number>2</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row/>
           <row/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ControlLayout" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="LeftControlGroup">
          <property name="title">
           <string>Graph Edit</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLineEdit" name="NodeNameEdit">
             <property name="placeholderText">
              <string>Enter Element's Name</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="AddDelLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QPushButton" name="addNodeButton">
               <property name="styleSheet">
                <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteNodeButton">
               <property name="styleSheet">
                <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="desButton">
             <property name="styleSheet">
              <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
             </property>
             <property name="text">
              <string>Cancel Orientation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddEdge">
             <property name="styleSheet">
              <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
             </property>
             <property name="text">
              <string>Add Connection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DeleteEdgeButton">
             <property name="styleSheet">
              <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
             </property>
             <property name="text">
              <string>Delete Connection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ClearButton">
             <property name="styleSheet">
              <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="matrButton">
             <property name="styleSheet">
              <string notr="true">font: 85 8pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
             </property>
             <property name="text">
              <string>Build Matrix</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="RightContolLayoyt" stretch="0">
          <item>
           <widget class="QGroupBox" name="ImageControlGroup">
            <property name="title">
             <string>Graph Parameters</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="weighBox">
               <property name="text">
                <string>Connection Weights</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="FlowBox">
               <property name="text">
                <string>Show Flows</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="gravityBox">
               <property name="text">
                <string>Gravity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton">
               <property name="styleSheet">
                <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 18, 0, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(41, 158, 9, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(26, 202, 54, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(23, 156, 51, 255), stop:0.935 rgba(0, 176, 0, 255), stop:1 rgba(35, 40, 3, 255));
font: 85 8pt &quot;Arial Black&quot;;</string>
               </property>
               <property name="text">
                <string>Randomize Colors</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="shuffleButton">
               <property name="styleSheet">
                <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 18, 0, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(41, 158, 9, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(26, 202, 54, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(23, 156, 51, 255), stop:0.935 rgba(0, 176, 0, 255), stop:1 rgba(35, 40, 3, 255));
font: 85 8pt &quot;Arial Black&quot;;</string>
               </property>
               <property name="text">
                <string>Randomize Positions</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="save_pos_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="help_action"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="save_pos_action">
   <property name="text">
    <string>Save with positions</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="help_action">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
